pom.xml

```diff
@@ -275,7 +275,8 @@
             <plugin>
                 <groupId>org.sonatype.plugins</groupId>
                 <artifactId>nexus-staging-maven-plugin</artifactId>
-                <version>1.6.10</version>
+                <!-- Latest version 1.6.10 is broken: https://issues.sonatype.org/projects/NEXUS/issues/NEXUS-31214 -->
```

Comment:
```suggestion
                <!-- Version 1.6.10 is broken: https://issues.sonatype.org/projects/NEXUS/issues/NEXUS-31214 -->
```
Can't say how long this is latest, so better not mention it. :-)

Reply:
😆